docs/source/guide/explanation/additional_features/configurable_input_size.rst

```diff
@@ -0,0 +1,65 @@
+Configurable Input Size
+=======================
+
+The Configurable Input Size feature allows users to adjust the input resolution of their deep learning models
+to balance between training and inference speed and model performance.
+This flexibility enables users to tailor the input size to their specific needs without manually altering
+the data pipeline configurations.
+
+To utilize this feature, simply specify the desired input size as an argument during the train command.
+Additionally, OTX ensures compatibility with model trained on non-default input sizes by automatically adjusting
+the data pipeline to match the input size during other engine entry points.
+
+Usage example:
+
+.. code-block::
+
+    $ otx train \
+        --config ... \
+        --data.input_size 512
```

Comment:
Couldn't we provide the same example via the API? How about adding API examples as well?

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3870/commits/400bd294597d4102c5fd414376c0d4859918c97f